go.mod

```diff
@@ -21,6 +21,7 @@ require (
 	github.com/gogo/googleapis v1.4.0 // indirect
 	github.com/heroku/docker-registry-client v0.0.0-20190909225348-afc9e1acc3d5
 	github.com/imdario/mergo v0.3.9
+	github.com/iwilltry42/confd v0.16.1 // indirect
```

Comment:
I guess you used this for testing (`go get`?)
Let's remove this from here and then run `go mod tidy` and `go mod vendor` to remove it from the other dep files :+1:

Reply:
@iwilltry42 thatnks for review. Rebased.